src/main/java/frc/robot/util/SwerveRequestFieldCentric.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.util;
+
+import com.ctre.phoenix6.StatusCode;
+import com.ctre.phoenix6.mechanisms.swerve.SwerveModule;
+import com.ctre.phoenix6.mechanisms.swerve.SwerveRequest.FieldCentric;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+
+public class SwerveRequestFieldCentric extends FieldCentric {
```

Comment:
This name doesn't tell me why it's extended off of FieldCentric. The only thing that changed is the optimize states I think, so the name should reflect that.